clients/javascript_client/README.md

```diff
@@ -1 +1,15 @@
 # Javascript Client for FMA service
+An api client wrapper for connecting to the federated-learning-service.
+
+## Setup
+Please run the ```setup``` at the top level of the directory
```

Comment:
```suggestion
Please run the the following command at the top level of the repo
```
currently which directory is ambiguous...  is it the `clients/` or `javascript_client` or entire repo?